src/argilla/client/login.py

```diff
@@ -0,0 +1,77 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from pathlib import Path
+from typing import Dict, Optional
+
+from pydantic import AnyHttpUrl, BaseModel
+
+from argilla.client.api import init
+from argilla.client.sdk.commons.errors import UnauthorizedApiError
```

Comment:
Related to a comment mentioned below:

```suggestion
from argilla.client.sdk.commons.errors import HttpResponseError, UnauthorizedApiError
```